tests/integration_tests/syncpoint/conf/changefeed.toml

```diff
@@ -0,0 +1,2 @@
+enable-sync-point = true
+sync-point-interval = "30s"
```

Comment:
How is the "30s" converted to time.Duration? Can the user write this as '30sec' or '30 sec' or '30 s'?

Reply:
no, the doc will tell the users they can only use format like: "3h4m30s"

Reply:
We can keep this configuration item similar as `consistent.flush-interval` (it is in millisecond) and write it as follows:

```
sync-point-interval = 30
```